src/library/layout2.jl

```diff
@@ -67,7 +67,7 @@ end
 
 render(ps::Pages, state) =
     Elem("iron-pages",
-        map(t -> Elem("section", render(t, state)), ps.pages.tiles),
+        map(t -> Elem("div", render(t, state)), ps.pages.tiles),
```

Comment:
It is not required, the current docs for iron-pages indicate that these should be divs. I'd assume that  selection would work, but I went with consistency. 

https://elements.polymer-project.org/elements/iron-pages